magpie/models.py

```diff
@@ -443,7 +443,7 @@ def url(self, settings=None):
 
     def expired(self):
         expire = get_constant("MAGPIE_TOKEN_EXPIRE", raise_missing=False, raise_not_set=False, default_value=86400)
-        return (datetime.datetime.utcnow() - self.created) <= datetime.timedelta(seconds=expire)
+        return (datetime.datetime.utcnow() - self.created) > datetime.timedelta(seconds=expire)
```

Comment:
Now that webhook test chain is complete, can we have a quick test for this?
Just use a small delta to ensure it triggers and validate the app returns the appropriate HTTP response.

Reply:
I am not sure to understand here what would be required.
Is it to test the webhook chain that uses a temporary token? 
There is the test `test_Webhook_CreateUser_FailingWebhook` which tests the token, by simulating an error in the webhook and calling the token's url back. It also uses the expired function in the background.

Or is it a new test I should add that tests the expired() function explicitly?

Reply:
To speed up this PR, we will add it as another PR since not explicitly needed for the webhook feature.